airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.json

```diff
@@ -83,6 +83,12 @@
             }
           }
         ]
+      },
+      "unset_dynamic_fields": {
+        "type": "boolean",
+        "description": "Set 'hs_time_in_*' properties to 'null'.",
+        "default": false,
```

Comment:
It means that we need to update our test credentials and set this option to True

Reply:
yes in GSM variable SECRET_SOURCE-HUBSPOT_CREDS

{
  "start_date": "2021-01-10T00:00:00Z",
  "credentials": {
    "credentials_title": "API Key Credentials",
    "api_key": "****************"
  },
  "unset_dynamic_fields": true
}